### DISCUSSION TITLE
[Feature request] "p" midfix for "per" is not intuitive and possibly misleading

### DISCUSSION THREAD

**Discussion by EdSchouten** [grafana/grafana#36985] [Dashboards, Panels & Visualizations]
- What Grafana version are you using?

4.1.0

- What datasource are you using?

Prometheus

- What OS are you running grafana on?

A recent Debian system, running Grafana in a container using this image: https://github.com/grafana/grafana-docker

- What's the bug?

I've noticed that the throughput unit definitions of Grafana have some abbreviations that are not intuitive. For example, for "throughput > reads/sec", I would expect the axis of the graph to show "rd/s". For writes, "wr/s" would be the most intuitive. Instead, it uses "rps" and "wps". Though I get where this comes from ("mph", as used in the US), "rps" and "wps" are not commonly used. "rps" typically means "rounds per second".

The reason I'm bringing this up is because for "operations per second", Grafana uses the abbreviation "ops". I've noticed that most of the users of my dashboards interpret this as "operations", meaning that they don't understand that the graph corresponds to a rate.

I would hereby like to propose that the abbreviations are changed to "rd/s", "wr/s", "op/s", etc.

**Comment by mitjaziv** [comment]
I thinks rd/s and wr/s are not really, the right units. 

What do you think about this:
<img width="126" alt="screen shot 2017-03-07 at 12 48 48" src="https://cloud.githubusercontent.com/assets/10356476/23655643/14069916-0336-11e7-9932-e74548b5b037.png">

<img width="87" alt="screen shot 2017-03-07 at 12 59 51" src="https://cloud.githubusercontent.com/assets/10356476/23655649/1db0d652-0336-11e7-8a1c-77510e539aa9.png">

I have a change which i can push, if you like it.

Regards

**Comment by EdSchouten** [comment]
That looks perfectly fine to me. Thanks!

**Comment by mitjaziv** [comment]
pull request for this feature https://github.com/grafana/grafana/pull/7751

**Comment by pkolyvas** [comment]
For those willing to pick up the issues, see the discussion on https://github.com/grafana/grafana/pull/7751 for some quick guidance on how this might be approached.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
